### DISCUSSION TITLE
beta66 command line launch not working

### DISCUSSION THREAD

**Discussion by cfritze64** [johnlindquist/kit#52] [Errors] [answered]
osX 11.2.3
Home is Users/BikeFixe

.profile in my home directory contains the line:
export PATH="$PATH:/Users/BikeFixe/.kenv/bin"

if I open Terminal and offer the command
 
kit

I get 

-bash: kit: command not found

If instead I offer 

/Users/BikeFixe/.kenv/bin/kit

scriptkit runs as expected. Launching my own scripts by their names not working either unless I offer the full path. What am I missing?

**Comment by johnlindquist** [comment, marked as answer]
@cfritze64 
Are you using bash or zsh? (run `echo $0` in your terminal)
If you're running zsh, that line should be added to your `~/.zshrc`. You may need to `touch ~/.zshrc` if you've never created one.

Also, make sure to restart your terminal, or `source .zshrc`, after adding to your PATH so that it picks up the changes.

**Comment by cfritze64** [comment]
using bash
Good golly: simply restarting Terminal as you suggested fixed it. Thx
